src/App.js

```diff
@@ -1,27 +1,20 @@
-import logo from './logo.svg';
-import './App.css';
+import React, { Component } from 'react';
+import Calculator from './Components/Calculator';
+import './css/App.css';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          <code>src/App.js</code>
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <main className="container">
+        <Calculator />
+      </main>
+    );
+  }
 }
```

Comment:
- [ ] Kindly make sure to use the functional components in order to meet the project requirement. :wink: 
    
   ![image](https://user-images.githubusercontent.com/87197412/218887662-67a68405-5f23-4545-919b-c90aad693a02.png)